playbooks/molecule/rocky8_omero/molecule.yml

```diff
@@ -63,12 +64,34 @@ platforms:
       - 443
       - 4063
       - 4064
+      - 4080
     published_ports:
-      - 127.0.0.1:4063:4063
-      - 127.0.0.1:4064:4064
       - 127.0.0.1:8080:80
     etc_hosts:
       omero.db.local: 192.168.56.2
+      omero.mail.local: 192.168.56.4
+
+  - name: omero_mail
+    hostname: omero.mail.local
+    image: rnwood/smtp4dev
+    required: true
+    command:
+    cgroupns_mode: host
+    privileged: true
+    pre_build_image: true
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:rw
+      - ${MOLECULE_SCENARIO_DIRECTORY}/../resources/shared/resolv.conf:/etc/resolv.conf
+      - ${MOLECULE_SCENARIO_DIRECTORY}/smtp-app-settings.json:/smtp4dev/appsettings.json
```

Comment:
we should be able to set environment variables for `smtp4dev`. I tried adding some env vars to an `env` block for the provisioner, but it seems molecule only uses these when creating the containers, and doesn't set the env vars inside the containers. So instead I've added the settings to a file and mount it in the container